### ISSUE TITLE
Accessibility WCAG 2.1 AA (including ARIA tags and skipto.js)

### ISSUE THREAD

**Issue by angrave** [ncsa/lumen#3]
Web pages should be digitally accessible i.e. 

- WCAG 2.1 AA Compliant
- Use ARIA tags correctly
- Use skipto.js.
- Add  instructions (e.g. CLAUDE.md) to include accessibility hints to ensure future HTML code meets WCAG 2.1 AA requirements.

(I'm about to create a PR for this)
